### PR TITLE
Applications: nrf5340_audio: Oct 2986 split vol ctrl and vol rend

### PR DESCRIPTION
**Pull request by koffes**

### CODE REVIEW

**Comment by koffes** [comment]
Just renaming, splitting and moving of files. No functional changes.

File: applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/CMakeLists.txt [hunk 1]
```diff
@@ -0,0 +1,26 @@
+#
+# Copyright (c) 2023 Nordic Semiconductor
```
Comment (alexsven) [inline applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/CMakeLists.txt:2]: Nit: Very thin line between this being a new file and a changed version of the previous one. Might update the year if this is considered a new file

**Review by gWacey — APPROVED** [review]

File: applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_ctlr_internal.c
Comment (alexsven) [inline applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_ctlr_internal.c:1]: We haven't used the `internal` keyword for any other c-files, so I suggest we only use it for header files

File: applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_rend_internal.h [hunk 2]
```diff
@@ -0,0 +1,66 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#ifndef _BT_VOL_REND_INTERNAL_H_
+#define _BT_VOL_REND_INTERNAL_H_
+
+#include <zephyr/bluetooth/conn.h>
```
Comment (alexsven) [inline applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_rend_internal.h:10]: This is not needed in this file

File: applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/bt_rendering_and_capture.c [hunk 3]
```diff
@@ -102,13 +126,12 @@ int bt_rend_volume_unmute(void)
 	return ret;
 }
 
-int bt_rend_discover(struct bt_conn *conn)
+int bt_r_and_c_discover(struct bt_conn *conn)
 {
 	int ret;
 
-	/* Only do a VCS discover if we are volume controller */
 	if (IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
```
Comment (alexsven) [inline applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/bt_rendering_and_capture.c:133]: Should there be an else here and a warning print that Discover is not supported?

File: applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_ctlr_internal.c [hunk 4]
```diff
@@ -0,0 +1,262 @@
...
+		return index;
+	}
+
+	ret = bt_vcp_vol_ctlr_discover(conn, &vcs_client_peer[index]);
+	return ret;
+}
+
+int bt_vol_ctlr_init(void)
+{
+	if (!IS_ENABLED(CONFIG_BT_VCP_VOL_CTLR)) {
```
Comment (alexsven) [inline applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_ctlr_internal.c:250]: This file is not compiled unless `CONFIG_BT_VCP_VOL_CTLR` is enabled, so we don't need this check anymore. I guess someone could manually add this c-file, but then they are on their own...

File: applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_rend_internal.c [hunk 5]
```diff
@@ -0,0 +1,138 @@
...
+	if (err) {
+		LOG_ERR("VCS flag callback error: %d", err);
+	} else {
+		LOG_DBG("Volume flags = 0x%01X", flags);
+	}
+}
+
+int bt_vol_rend_set(uint8_t volume)
+{
+	if (!IS_ENABLED(CONFIG_BT_VCP_VOL_REND)) {
```
Comment (alexsven) [inline applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/volume/bt_vol_rend_internal.c:66]: I think we can remove all of the checks of this type in this file. The file is not compiled unless `CONFIG_BT_VCP_VOL_REND`is enabled

**Review by alexsven — CHANGES_REQUESTED** [review]

File: applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/CMakeLists.txt (hunk 1 above)
Comment (koffes) [reply applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/CMakeLists.txt:2]: Most of the content is old so opt to keep as is.

File: applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/bt_rendering_and_capture.c (hunk 3 above)
Comment (koffes) [reply applications/nrf5340_audio/src/bluetooth/bt_rendering_and_capture/bt_rendering_and_capture.c:133]: Added some error returns.

**Review by alexsven — APPROVED** [review]

**Pull request merged**